src/Security/SamlBundle/DashboardSamlBundle.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of letswifi; a system for easy eduroam device enrollment
+ * Copyright: 2023, Paul Dekkers, SURF <paul.dekkers@surf.nl>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+namespace App\Security\SamlBundle;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class DashboardSamlBundle extends Bundle
```

Comment:
Vermoed dat deze file wegkan, je hebt dit al in de vendors ergens staan gok ik :)

Reply:
Eigenlijk alles wat onder SamlBundle staat kan weg denk ik

Reply:
Dit is het stukje implementatie client side en nodig voor authenticatie.

Reply:
Uiteindelijk heb je alleen de provider een een controller voor je metadata nodig kwamen we gisteren achter.

Reply:
Klopt, haal m maar meteen weg.